### PR TITLE
basketballking.jp

### PR DESCRIPTION
**Pull request by SKEIDs**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [x] Missed analytics or tracker;
- [x] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

https://basketballking.jp/news/japan/b1/20221213/406872.html?cx_art=rankingcat

### Add your comment and screenshots

tracking parameter is from side

<details><summary>screenshot</summary>

</details><br/>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -9,7 +9,7 @@
...
...arweb.com,singstar.com,ski.com.au,skidkaonline.ru,skidrowcrack.com,skype.com,slantmagazine.com,sltrib.com,snp.org,soulwire.co.uk,sourcefednews.com,spankbang.com,speakingtree.in,sportfm.ru,sportingnews.com,springer.com,sprintfeed.com,starbucks.com,stars-portraits.com,steampowered.com,suntimes.com,surfacemag.com,syl.ru,taiwantoday.tw,take40.com,tanisgazetesi.com.tr,teamtalk.com,technobuffalo.com,ted.com,temp-mail.ru,texterra.ru,thedailymeal.com,thefile.me,theglobaldispatch.com,thenational.ae,theringreport.com,thewindowsclub.com,timeline.com,timeslive.co.za,todoanimeshd.com,tomonews.net,tor.com,torontosun.com,total-croatia-news.com,tunein.com,tv.yahoo.com,tvnz.co.nz,twitchy.com,uanews.org,unhearit.com,upmedia.mg,urbandictionary.com,usatoday.com,utoronto.ca,van.man,vanityfair.com,vectorstock.com,vellisa.ru,verisign.com,versia.ru,vesti-ukr.com,vgtimes.ru,vh1.com,video.infonet.vn,videoweed.es,virtex.com,visiontimes.com,vulture.com,wanda-group.com,washingtonpost.com,wegotthiscovered.com,wheels24.co.za,winnipegfreepress.com,wired.co.uk,wmagazine.com,wrc.com,wsj.com,xarold.com,xda-developers.com,xhamster.com,xhamster.one,xxxymovies.com,yeniasir.com.tr,yipit.com,zaochnik.ru,zapak.com##.share
```
Comment (Yuki2718) [inline SocialFilter/sections/specific.txt:12]: I don't see `##.share` on the page.

File: SocialFilter/sections/specific.txt [hunk 2]
```diff
@@ -3496,7 +3496,7 @@ babyclub.de##.social-links-container
 censor.net##.footer_theme
 diy.obi.ru##.rating-section-share
 edaily.co.kr##.reporter_sns
-autos.goo.ne.jp,ch-review.net,kuruma-news.jp,alba.co.jp,mskagency.ru,pr.goo.ne.jp,point.md,bike-news.jp,haber7.com,w.atwiki.jp,badische-zeitung.de##.social-button
+basketballking.jp,ruma-news.jp,alba.co.jp,mskagency.ru,pr.goo.ne.jp,point.md,bike-news.jp,haber7.com,w.atwiki.jp,badische-zeitung.de##.social-button
```
Comment (Yuki2718) [inline SocialFilter/sections/specific.txt:3499]: Mistakenly removed?

File: TrackParamFilter/sections/specific.txt [hunk 3]
```diff
@@ -9,6 +9,8 @@
 ! https://github.com/AdguardTeam/AdguardFilters/issues/136734
 ||investing.com^$cookie=smd
 ||investing.com^$cookie=udid
+! https://github.com/AdguardTeam/AdguardFilters/pull/137576
+||basketballking.jp^$removeparam=cx_art
```
Comment (Yuki2718) [inline TrackParamFilter/sections/specific.txt:13]: @Alex-302 @zloyden IMO this should be generic and added to `! Cxense clickthrough tracking` line.

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

File: SocialFilter/sections/specific.txt (hunk 1 above)
Comment (SKEIDs) [reply SocialFilter/sections/specific.txt:12]: it is "ツイート" at the comment area.

File: SocialFilter/sections/specific.txt (hunk 2 above)
Comment (SKEIDs) [reply SocialFilter/sections/specific.txt:3499]: sorry , i mistook that.

File: SocialFilter/sections/specific.txt (hunk 1 above)
Comment (Yuki2718) [reply SocialFilter/sections/specific.txt:12]: Okay I see, but I'm not sure whether that's worth hidding. Let's wait for others' review.

Comment (Yuki2718) [reply SocialFilter/sections/specific.txt:12]: Okay I see, but I'm not sure whether that's worth hidding. Let's wait for others' review.

File: SocialFilter/sections/specific.txt (hunk 2 above)
Comment (Yuki2718) [reply SocialFilter/sections/specific.txt]: Looks `autos.goo.ne.jp,` is still not recovered.

Comment (SKEIDs) [reply SocialFilter/sections/specific.txt]: sorry.

File: SocialFilter/sections/specific.txt (hunk 1 above)
Comment (SKEIDs) [reply SocialFilter/sections/specific.txt:12]: I have seen it removed in other issues before maybe. so I removed it as same.

File: TrackParamFilter/sections/specific.txt (hunk 3 above)
Comment (Alex-302) [reply TrackParamFilter/sections/specific.txt:13]: @Yuki2718 is it related to Cxense?
I see it on 4 sites only. 3 - with no traffic.

**Review by Alex-302 — APPROVED** [review]

File: TrackParamFilter/sections/specific.txt (hunk 3 above)
Comment (Yuki2718) [reply TrackParamFilter/sections/specific.txt:13]: > is it related to Cxense?

Supposedly. Let's see if it's used in more sites.

**Pull request merged**